Configurations/15-ios.conf

```diff
@@ -17,22 +17,22 @@ my %targets = (
         # It should be possible to go below iOS 6 and even add -arch armv6,
         # thus targeting iPhone pre-3GS, but it's assumed to be irrelevant
         # at this point.
-        CC               => "xcrun -sdk iphoneos cc",
```

Comment:
it is unclear if dropping things as xcrun -sdk iphoneos is a good idea.

Reply:
> it is unclear if dropping things as xcrun -sdk iphoneos is a good idea.

Commit updated, `xcrun -sdk iphoneos cc` is a wrong command.